core/common/src/IModel.ts

```diff
@@ -68,6 +68,9 @@ export interface EcefLocationProps {
   readonly xVector?: XYZProps;
   /** Optional Y column vector used with [[xVector]] to calculate potentially non-rigid transform if a projection is present. */
   readonly yVector?: XYZProps;
+  /** Optional transform from which all other props will be computed - potentially non-rigid transform if a projection is present. */
```

Comment:
Update the interface documentation above "If the xVector and yVector properties are define then they take precedence..."

Reply:
Fix pushed